{
  "url": "https://n8n.io/workflows/8-handle-errors-from-a-different-workflow/",
  "title": "Handle errors from a different workflow",
  "author": "Jan Oberhauser",
  "publish_date": "Last update 10 months ago",
  "publish_date_absolute": "2024-07-25",
  "categories": [
    {
      "name": "Building Blocks"
    }
  ],
  "workflow_json": "{\"nodes\":[{\"name\":\"Error Trigger\",\"type\":\"n8n-nodes-base.errorTrigger\",\"position\":[250,500],\"parameters\":{},\"typeVersion\":1},{\"name\":\"Mailgun\",\"type\":\"n8n-nodes-base.mailgun\",\"position\":[450,500],\"parameters\":{\"text\":\"=Error: {{$node[\\\"Error Trigger\\\"].data[\\\"execution\\\"][\\\"error\\\"][\\\"message\\\"]}}\\n\\nStack Trace:\\n{{$node[\\\"Error Trigger\\\"].data[\\\"execution\\\"][\\\"error\\\"][\\\"stack\\\"]}}\",\"subject\":\"=Workflow Error:  {{$node[\\\"Error Trigger\\\"].data[\\\"workflow\\\"][\\\"name\\\"]}}\",\"toEmail\":\"\",\"fromEmail\":\"\"},\"credentials\":{\"mailgunApi\":\"\"},\"typeVersion\":1}],\"connections\":{\"Error Trigger\":{\"main\":[[{\"node\":\"Mailgun\",\"type\":\"main\",\"index\":0}]]}}}",
  "readme": "When set as \"Error Workflow\" on other workflow which does fail will it send an Email with information about which workflow did fail and what went wrong.\n",
  "crawled_at": "2025-05-25T17:44:35.233155"
}